toxcore/group_chats.c

```diff
@@ -156,6 +156,50 @@ non_null() static size_t load_gc_peers(GC_Chat *chat, const GC_SavedPeerInfo *ad
 non_null() static bool saved_peer_is_valid(const GC_SavedPeerInfo *saved_peer);
 
 static const GC_Chat empty_gc_chat = {nullptr};
+static const uint8_t empty_passkey[GC_PASSKEY_SIZE] = {0};
+
+
+/* We re-use the Group ID as salt for the password, if this condition is violated and out-of-bounds read happens */
+static_assert(EXT_PUBLIC_KEY_SIZE >= crypto_pwhash_scryptsalsa208sha256_SALTBYTES,
```

Comment:
afaik, toxcore uses C99, but static_assert is a C11 feature.
https://en.cppreference.com/w/c/error/static_assert

Reply:
It's used a bit above, so I think it works. Not sure though why...